### DISCUSSION TITLE
Progress Report 13–17 June

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#36] [Progress Report]
🚴🏻 **Progress**

- Implemented preliminary API for fetching temperature diagnostics

😖 **Regressions**

None

💬 **To Discuss**

- #35 
- #26 

📆 **Next Week**

- Visualize temperature diagnostics
- Prepare agenda for demo on 27 June

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**

- Added a CI job to keep the VLab image registry clean and reduce costs: https://github.com/NOAA-GSL/unified-graphics/pull/34
- Kubernetes manifest location: https://github.com/NOAA-GSL/unified-graphics/discussions/31#discussioncomment-2959302

😖 **Regressions**

- https://github.com/NOAA-GSL/unified-graphics/pull/37

💬 **To Discuss**

- None

📆 **Next Week**

- Work with VLab to get `main` images into the cluster
- Work on infrastructure for integration tests
- Prepare agenda for demo on 27 June
